src/services/webclient.ts

```diff
@@ -1752,18 +1752,29 @@ export class WebClientService {
 
                         // check receiver
                         switch (receiver.type) {
-                            case 'distributionList':
-                                return reject(this.$translate.instant(
-                                    invalidFeatureMessage, {receiverName: receiver.displayName},
-                                ));
                             case 'group':
+                            case 'distributionList':
                                 const unsupportedMembers = [];
-                                const group = this.groups.get(receiver.id);
-
-                                if (group === undefined) {
-                                    return reject();
+                                let members: string[];
+                                switch (receiver.type) {
+                                    case 'group':
+                                        const group = this.groups.get(receiver.id);
+                                        if (group === undefined) {
+                                            this.log.error(`Group ${receiver.id} not found`);
+                                            return reject(this.$translate.instant('error.ERROR_OCCURRED'));
+                                        }
+                                        members = group.members;
+                                        break;
+                                    case 'distributionList':
+                                        const distributionList = this.distributionLists.get(receiver.id);
+                                        if (distributionList === undefined) {
+                                            this.log.error(`Distribution list ${receiver.id} not found`);
+                                            return reject(this.$translate.instant('error.ERROR_OCCURRED'));
+                                        }
+                                        members = distributionList.members;
+                                        break;
                                 }
-                                group.members.forEach((identity: string) => {
+                                members.forEach((identity: string) => {
```

Comment:
Any reason to not make this is a normal `for`?

Reply:
I don't think so. I'll change it.